tests/functional/test_e2e.py

```diff
@@ -324,8 +324,8 @@ def test_e2e_release_files(
     find_and_click(page.locator("#outstanding-requests").get_by_role("link"))
     expect(page.locator("body")).to_contain_text(request_id)
 
-    complete_review_button = page.locator("#complete-review-button")
-    # output checker hasn't reviewed files yet, complete review button visible but disabled
+    complete_review_button = page.locator("#submit-review-button")
```

Comment:
```suggestion
    submit_review_button = page.locator("#submit-review-button")
```
and all the use of this var...